doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -99,17 +95,21 @@ The following client-side profile is based on the rules set in the schema.
 }
 ```
 
-The schema structure from our example allows filtering the endpoints by the operation system of device (for example, to show only Android devices), os_version (for example, to push some notifications only for the specified os version).
-You can create complex filtering conditions by combining as many filtering conditions as needed.
+The schema structure from this example allows filtering the endpoints by the operation system of device (for example, to show only Android devices), OS version (for example, to push some notifications only for the specified os version).
+You can create complex filters by combining as many filtering conditions as needed.
 
-### Setting client-side EP profile schema from Administration UI
+### Setting client-side EP profile schema
 
-To view the list of client-side endpoint profile schemas created by a [tenant developer]({{root_url}}Glossary/#tenant-developer), open the the **Client-side EP profile** page under the **Schemas** section of the application.
+To view the list of CT schemas used for the client-side endpoint profile, log in to [Administration UI]({{root_url}}Glossary/#administration-ui) and click **Tenant CTL**.
 
-<img src="admin-ui/Client-side endpoint profile schema.png">
+To export a CT schema:
 
-To export a client-side EP profile schema, click the corresponding **Export** button and choose the export method from the drop-down list.
-For more information about the schema export methods, see [CT schema export support]({{root_url}}Programming-guide/Key-platform-features/Common-Type-Library/#ct-schema-export-support).
+1. Log in to the Administration UI as a tenant developer using the default **devuser** username and **devuser123** password.
```

Comment:
Please, **never** specify default username / password in any pages except of the Sandbox intro. This user/pass combo **only** applies to the Sandbox.